Ledger_iOS/Models/Entry.swift

```diff
@@ -5,14 +5,31 @@
 //  Created by Elisha Chirchir on 20/12/2021.
 //
 
-import Foundation
 import RealmSwift
 
 class Entry: Object, ObjectKeyIdentifiable {
-    @Persisted(primaryKey: true) var id: ObjectId
-    @Persisted var balance: Balance? = Balance.init()
-    @Persisted var runningTotal: Float = 0.0
-    @Persisted var desc: String = ""
-    @Persisted var amount: Float = 0.0
-    @Persisted var date: Date?
+    @objc dynamic var id: String
+    @objc dynamic var projectId: String
+    @objc dynamic var itemId: String
+    @objc dynamic var incomeExpense: IncomeExpense
+    @objc dynamic var title: String = ""
+    @objc dynamic var desc: String = ""
+    @objc dynamic var createdDate: Date?
+    @objc dynamic var lastEditDate: Date?
+    @objc dynamic var numberOfUnits: Float
+    @objc dynamic var typeOfUnit: UnitType
```

Comment:
Why are we switching back to the @objc dynamic anotation? @Persisted should be newer I think?

Reply:
After looking at the documentation you are correct, ill change them back

Reply:
Also note to self: Create additional realm models specifically for realm and models for UI